app/assets/stylesheets/panel/panel.tickets.create.scss

```diff
@@ -0,0 +1,14 @@
+.create {
```

Comment:
We will move this into a regular panel.scss file. Create a .panel class, and within it create &.panel. This will create stylings for .panel.create and inside you can then hash out even more classes such as .top or .form { &.top{} &.label {} }.